api/internal/plugins/execplugin/execplugin_test.go

```diff
@@ -107,22 +103,23 @@ func TestExecPlugin_ErrIfNotExecutable(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
+	for _, obj := range srcRoot {
```

Comment:
For this one, it is looking for a root dir that is part of Kustomize's own test setup, so I _think_ we should instead be able to throw an error if the length of srcRoot is > 1 and avoid the loop. Given the purpose of the test, I don't think it makes sense to be checking multiple places.